acs-integration-tests/src/test/java/com/ge/predix/integration/test/PolicyEvaluationWithAttributeConnectorIT.java

```diff
@@ -0,0 +1,162 @@
+import com.ge.predix.acs.rest.AttributeAdapterConnection;
+import com.ge.predix.acs.rest.AttributeConnector;
+import com.ge.predix.acs.rest.PolicyEvaluationRequestV1;
+import com.ge.predix.acs.rest.PolicyEvaluationResult;
+import com.ge.predix.acs.rest.Zone;
+import com.ge.predix.test.utils.ACSRestTemplateFactory;
+import com.ge.predix.test.utils.PolicyHelper;
+import com.ge.predix.test.utils.UaaTestUtil;
+import com.ge.predix.test.utils.ZacTestUtil;
+import com.ge.predix.test.utils.ZoneHelper;
+
+@ContextConfiguration("classpath:integration-test-spring-context.xml")
+public class PolicyEvaluationWithAttributeConnectorIT extends AbstractTestNGSpringContextTests {
+
+    @Value("${ZONE1_NAME:testzone1}")
+    private String acsZone1Name;
+
+    @Value("${ZONE2_NAME:testzone2}")
+    private String acsZone2Name;
+
+    @Value("${ZONE3_NAME:testzone3}")
+    private String acsZone3Name;
+
+    @Value("${ACS_UAA_URL}")
+    private String acsUaaUrl;
+
+    @Autowired
+    private Environment environment;
+
+    @Autowired
+    private ACSRestTemplateFactory acsRestTemplateFactory;
+
+    @Autowired
+    private ZacTestUtil zacTestUtil;
+
+    @Autowired
+    private ZoneHelper zoneHelper;
+
+    @Autowired
+    private PolicyHelper policyHelper;
+
+    private static final String IDENTIFIER = "part/03f95db1-4255-4265-a509-f7bca3e1fee4";
+
+    private Zone zone;
+    private OAuth2RestTemplate acsAdminRestTemplate;
+    private boolean registerWithZac;
+    private URI resourceAttributeConnectorUrl;
+
+    private HttpHeaders getHeadersWithZoneId() throws IOException {
+        HttpHeaders httpHeaders = new HttpHeaders();
+        httpHeaders.set(PolicyHelper.PREDIX_ZONE_ID, this.zone.getSubdomain());
+        return httpHeaders;
+    }
+
+    private void configureAttributeConnector() throws IOException {
+        AttributeConnector attributeConnector = new AttributeConnector();
+        attributeConnector.setIsActive(true);
+        attributeConnector.setMaxCachedIntervalMinutes(60);
+        attributeConnector.setAdapters(new HashSet<>(Collections.singletonList(
+                new AttributeAdapterConnection("https://acs-asset-adapter.run.aws-usw02-dev.ice.predix.io/v1/attribute",
+                        "https://22738a33-40d6-4052-bc96-b9807688ec59.predix-uaa.run.aws-usw02-dev.ice.predix"
+                                + ".io/oauth/token", "connector", "connector"))));
+        this.resourceAttributeConnectorUrl = URI.create(this.zoneHelper.getAcsBaseURL() + AcsApiUriTemplates.V1
+                + AcsApiUriTemplates.RESOURCE_CONNECTOR_URL);
+        HttpHeaders httpHeaders = this.getHeadersWithZoneId();
+        httpHeaders.setContentType(MediaType.APPLICATION_JSON);
+        this.acsAdminRestTemplate.exchange(this.resourceAttributeConnectorUrl, HttpMethod.PUT,
+                new HttpEntity<>(attributeConnector, httpHeaders), AttributeConnector.class);
+    }
+
+    private void setupPredixAcs() throws IOException {
```

Comment:
I haven't refactored this yet but we can as part of this story or maybe another one?